scanners/ssh-scan/README.md

```diff
@@ -1,20 +1,106 @@
-# ssh-scan
+---
+title: "SSH"
+category: "scanner"
+type: "SSH"
+state: "released"
+appVersion: "0.0.44"
+usecase: "SSH Configuration and Policy Scanner"
+---
 
-![Version: v2.7.0-alpha1](https://img.shields.io/badge/Version-v2.7.0--alpha1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.0.44](https://img.shields.io/badge/AppVersion-0.0.44-informational?style=flat-square)
+<!--
+SPDX-FileCopyrightText: 2020 iteratec GmbH
 
-A Helm chart for the SSH_Scan security scanner that integrates with the secureCodeBox.
+SPDX-License-Identifier: Apache-2.0
+-->
+<!--
+.: IMPORTANT! :.
+--------------------------
+This file is generated automaticaly with `helm-docs` based on the following template files:
+- ./.helm-docs/templates.gotmpl (general template data for all charts)
+- ./chart-folder/.helm-docs.gotmpl (chart specific template data)
 
-**Homepage:** <https://docs.securecodebox.io/docs/scanners/SSH>
+Please be aware of that and apply your changes only within those template files instead of this file.
+Otherwise your changes will be reverted/overriden automaticaly due to the build process `./.github/workflows/helm-docs.yaml`
+--------------------------
+-->
 
-## Maintainers
+<p align="center">
+  <a href="https://opensource.org/licenses/Apache-2.0"><img alt="License Apache-2.0" src="https://img.shields.io/badge/License-Apache%202.0-blue.svg"></a>
+  <a href="https://github.com/secureCodeBox/secureCodeBox/releases/latest"><img alt="GitHub release (latest SemVer)" src="https://img.shields.io/github/v/release/secureCodeBox/secureCodeBox?sort=semver"></a>
+  <a href="https://owasp.org/www-project-securecodebox/"><img alt="OWASP Incubator Project" src="https://img.shields.io/badge/OWASP-Incubator%20Project-365EAA"></a>
+  <a href="https://artifacthub.io/packages/search?repo=seccurecodebox"><img alt="Artifact HUB" src="https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/seccurecodebox"></a>
+  <a href="https://github.com/secureCodeBox/secureCodeBox/"><img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/secureCodeBox/secureCodeBox?logo=GitHub"></a>
+  <a href="https://twitter.com/securecodebox"><img alt="Twitter Follower" src="https://img.shields.io/twitter/follow/securecodebox?style=flat&color=blue&logo=twitter"></a>
+</p>
 
-| Name | Email | Url |
-| ---- | ------ | --- |
-| iteratec GmbH | secureCodeBox@iteratec.com |  |
+## What is SSH_scan?
```

Comment:
```suggestion
## What is ssh_scan?
```

Reply:
I would suggest writing ssh in lower case because this would match the official repository